test/uk/gov/hmrc/eusubsidycompliancefrontend/services/EligibilityJourneySpec.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.services
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import uk.gov.hmrc.eusubsidycompliancefrontend.services.EligibilityJourney.FormUrls.{CustomsWaivers, EoriCheck, MainBusinessCheck, WillYouClaim}
+
+class EligibilityJourneySpec extends AnyWordSpecLike with Matchers {
+
+  "EligibilityJourney" when {
+
+    "steps is called" should {
+
+      "return all forms at the start of the journey" in {
+        val underTest = EligibilityJourney()
+        underTest.formPages shouldBe List(
```

Comment:
minor : We can make one LISt with all these values
underTest.customsWaivers,....
          underTest.willYouClaim,
 and use .filter or filterNot in other test w.r.t this LIst